synced-cron-server.js

```diff
@@ -225,11 +225,11 @@ SyncedCron._entryWrapper = function(entry) {
         }
       });
     } catch(e) {
-      log.info('Exception "' + entry.name +'" ' + e.stack);
+      log.info('Exception "' + entry.name +'" ' + ((e && e.stack) ? e.stack : e));
```

Comment:
It will be better:
`log.info('Exception "' + entry.name +'" ' + (e && e.stack || e));`